tests/multiprocessing_smoke_tests.py

```diff
@@ -29,7 +29,10 @@
            "get_value": get_test_values,
            "device": "cpu",
            "num_processes": N_ENVS,
-           "num_steps": 2048
+           "num_steps": 2048,
+           "robust_plr": True,
+           "eval_envs": create_nethack_env(),
+           "action_value_fn": get_action_value
```

Comment:
If you need this to output a single action and value for now, just do something simple for testing
```
def get_action_value(obs):
    return 0, 0
````